_posts/en/0500-10-24-tm-admin_en.md

```diff
@@ -137,11 +140,11 @@ After uploading a file to the Tasking Manager you can select if you want to defi
 1. `Square Grid`, with the functionality described above to define tasks.
 2. `Arbitrary Geometries`, coming from the file you just uploaded.
 
-#### Define arbitrary geometries
+**Define arbitrary geometries**
 
-If you uploaded a file to define your project area, that file can also contain the shapes for the individual tasks. E.g. if the area of interest consists of one polygon, the project will have just one task. This is generally not needed and should only be needed for specific needs. An import of roads project might need custom task shapes or the output from an mapping area reduction process for example.
+If you uploaded a file to define your project area, that file may also contain the shapes for the individual tasks. E.g. if the area of interest consists of one polygon, the project will have just one task. This is generally not needed and should only be needed for specific needs. An import of roads project might need custom task shapes or the output from an mapping area reduction process for example.
```

Comment:
Suggestion: 
If you uploaded a file to define your project area, that file may also contain the shapes for the individual tasks. E.g. if the area of interest consists of one polygon, the project will have just one task. This should only be used in special cases and for specific reasons. For example, an import of roads project might need custom task shapes or output from a mapping area reduction process